docker/test.sh

```diff
@@ -0,0 +1,22 @@
+# - VERY rudimentary test script to run latest + specific branch image builds and test them all by running `--version`
+# TODO: create a real test suite
+
+clear \
```

Comment:
This entire thing could be cleaned up by just setting `-e` using `setopts` at the top and then not having to `&&` and `\` every line.

Reply:
lol, probably right, this is copy/pasta from the original and I don't think it was ever used. I wouldn't be against removing it entirely since I don't think it is used anywhere. And if it is, the check can be replaced with a one-liner.